### PR TITLE
Feat/popover

### PR DESCRIPTION
**Pull request by darintrimillos**
Elizabeth has given a thumbs up, so here we go!

### CODE REVIEW

File: src/components/popover/CdrPopover.backstop.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = [{
+  label: 'Popover',
+  selectors: [],
```
Comment (lancetharper) [inline src/components/popover/CdrPopover.backstop.js:3]: No backstop tests? You can do clicks and stuff to launch the modal, but I don't remember exactly how to do that and we might need to wire that functionality up.

File: src/components/popover/CdrPopover.vue [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    hasDefaultStyling: {
+      type: Boolean,
+      required: false,
+      default: true,
+    },
+
+    /**
+     * Aria label - the popover title.
+     */
+    ariaLabelledBy: {
```
Comment (lancetharper) [inline src/components/popover/CdrPopover.vue:41]: If these aria attributes aren't required we don't really need props for them (unless we need the defaults for basic a11y). If they are added by consumers they will just get appended to the root element anyway.

File: src/components/popover/CdrPopover.backstop.js (hunk 1 above)
Comment (darintrimillos) [reply src/components/popover/CdrPopover.backstop.js:3]: I totally forgot about this. Will look into it.

Comment (mhewson) [reply src/components/popover/CdrPopover.backstop.js:3]: this is all for the pre v1 component correct?

**Pull request closed without merging**